packages/vda-node-manager/test/user_write.test.ts

```diff
@@ -39,6 +39,7 @@ describe('vda-node-manager read and write tests', () => {
         registeredCentreIds = await addInitialData(configuration);
     })
 
+    /*
```

Comment:
Remove

Reply:
It's used in the `Data Centre Test`.
Pushed a new commit